### DISCUSSION TITLE
Cannot insert backticks in JetBrains when Copilot is activated.

### DISCUSSION THREAD

**Discussion by persello** [community/community#11046] [Copilot Conversations]
When using JetBrains CLion and WebStorm (might also happen on other JetBrains IDEs), I can’t write backticks (`) in my code.

I use the Macintosh Italian keyboard layout, and since the “Copilot: Show completions” keybinding is <kbd>option</kbd>+<kbd>/</kbd>, it collides with the backtick key.

I think this might cause some confusion when not properly configured.

**Comment by jansorg** [comment]
@persello Thanks for reporting! Is it really `option + slash` or `option + backslash`? It should be the latter as default shortcut.
We had similar reports in the past, but have not yet found a good default for everyone.
What would be a better shortcut for you?

**Reply by persello** [reply]
Yes, sorry, I meant to say backslash.

I don't know what to suggest as a replacement key-binding, since many already collide without installing any other plugin.

Since I rely on what is automatically suggested by Copilot, I have never used that shortcut, so I can't really give you a valid alternative.
